ci/terraform/shared/dynamodb.tf

```diff
@@ -660,3 +660,35 @@ data "aws_iam_policy_document" "cross_account_identity_credentials_table_resourc
     resources = ["*"]
   }
 }
+
+resource "aws_dynamodb_table" "auth_session_table" {
+  name         = "${var.environment}-auth-session"
+  billing_mode = var.provision_dynamo ? "PROVISIONED" : "PAY_PER_REQUEST"
+
+  hash_key = "SessionId"
+
+  attribute {
+    name = "SessionId"
+    type = "S"
+  }
+
+  ttl {
+    attribute_name = "TimeToLive"
+    enabled        = true
+  }
+
+  point_in_time_recovery {
+    enabled = true
+  }
```

Comment:
We should probably think about if we want pitr on for this or not. It's session data, so I'm not sure if we would ever want to restore it or not

Reply:
Will make a ticket for it